helper-cli/src/main/kotlin/commands/StorageListCommand.kt

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (C) 2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.helper.commands
+
+import com.beust.jcommander.DynamicParameter
+import com.beust.jcommander.JCommander
+import com.beust.jcommander.Parameter
+import com.beust.jcommander.Parameters
+
+import com.here.ort.helper.CommandWithHelp
+import com.here.ort.helper.common.IdentifierConverter
+import com.here.ort.model.Identifier
+import com.here.ort.model.config.OrtConfiguration
+import com.here.ort.model.config.ScannerConfiguration
+import com.here.ort.model.yamlMapper
+import com.here.ort.scanner.ScanResultsStorage
+import com.here.ort.utils.PARAMETER_ORDER_MANDATORY
+import com.here.ort.utils.PARAMETER_ORDER_OPTIONAL
+
+import java.io.File
+
+@Parameters(
+    commandNames = ["storage-list"],
+    commandDescription = "Lists the provenance of all stored scan results for a given package identifier."
+)
+internal class StorageListCommand : CommandWithHelp() {
+    @Parameter(
+        description = "The path to the ORT configuration file that configures the scan results storage.",
+        names = ["--config"],
+        order = PARAMETER_ORDER_OPTIONAL
+    )
```

Comment:
This adds a dependency in the whole OrtConfiguration.
It can be easily avoided by having a `--storage-configuration parameter` pointing to either a
`PostgresStorageConfiguration.yml` or to a `FileBasedStorageConfiguration.yml`.
I'd prefer that since to keep things `decoupled` and to `minimize dependencies`, I do not foresee any other helper-cli commands needing the OrtConfiguration. What do you think about that?

Reply:
If you use a storage other than the default you already have a ORT config file which you can reuse here. Requiring a storage configuration YAML file here would mean you have to convert part of your configuration file to a different format before you can use it here.

Reply:
Yeah, I think this one time effort of setting up the YAML file is reasonable spend that file (1) contains only exactly what is needed and thus minimized complexity and (2) it keeps the helper-cli separate from any ort configuration....which I regard valuable.

Anyhow, I wouldn't see a big issue as long as we do not start introducing dependencies on the ORT configuration in the helper-cli all over the place, which I fear could happen which is why I'm a bit afraid. I'll leave this point up-to-you to decide.

Reply:
As a user I would find it very inconvenient if I had to convert the config file I use with `ort` to be able to use it with `orth`, that's why I prefer to keep it as is. I already use several of them for testing and wouldn't like to duplicate them and have to do future changes in two files instead of one.